content/Arduino Cloud/General Cloud/Supported-Wi-Fi-bands.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Supported Wi-Fi bands"
+id: 14144656051740
+---
+
+## Use 2.4 GHz Band for IoT Devices
```

Comment:
```suggestion
```
I would remove this header and use the sentence in line 8 as an introduction to the article.